src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:5.0-buster-slim
+ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:6.0-bullseye-slim
```

Comment:
Do we need this? Seems like buster is still latest stable...

Reply:
No we don't. I didn't realize it's "testing" and not "stable". I just took the newest without checking. I'll revert it.

Reply:
We'll need the bullseye after all. Seems like buster doesn't have current 6.0 preview SDK...